forest/src/cli/mod.rs

```diff
@@ -451,19 +463,19 @@ mod test {
         ];
 
         for (input, expected) in cases {
-            assert_eq!(to_size_string(&input), expected.to_string());
+            assert_eq!(to_size_string(&input).unwrap(), expected.to_string());
```

Comment:
Maybe something like this:

```suggestion
            assert_eq!(to_size_string(&input), Ok(expected.to_string()));
```